plugins/queue/smtp_forward.js

```diff
@@ -82,15 +90,19 @@ exports.hook_queue = function (next, connection) {
                     smtp_client.send_command('AUTH', 'PLAIN ' + base64('\0' + cfg.auth_user + '\0' + cfg.auth_pass));
                 }
                 else if (cfg.auth_type === 'login') {
+                    smtp_client.authenticating = true;
+                    smtp_client.authenticated=false;
+
+                    connection.loginfo(plugin, 'Authenticating with AUTH LOGIN ' + cfg.auth_user);
                     smtp_client.send_command('AUTH', 'LOGIN');
-                    smtp_client.on('auth', function () {
-                        connection.loginfo(plugin, 'Authenticating with AUTH LOGIN ' + cfg.auth_user);
+                    smtp_client.on('auth', function() {
```

Comment:
code style issue, please add space between function keyword and parens in function declarations.